cypress/steps/actions-page-steps.ts

```diff
@@ -44,19 +46,11 @@ export default class ActionsPageSteps {
   }
 
   static toggleIncludeInferred() {
-    cy.get('#toggleIncludeInferred').click();
-  }
-
-  static hideExpandResultsAction() {
-    cy.get('#hideExpandResultsAction').click();
-  }
-
-  static showExpandResultsAction() {
-    cy.get('#showExpandResultsAction').click();
+    YasqeSteps.getIncludeInferredStatementsButton().click();
```

Comment:
There is something wrong here. This YasqePageSteps class is intended for accessing only the stuff on the page but excluding the yasgui/yasqe/yasr.

Reply:
Fixed. Removed methods in correct step file.